src/encode.cpp

```diff
@@ -277,4 +278,48 @@ operator>>(MessageBuffer& buffer, PublishIntentEnd& msg)
   return buffer;
 }
 
+messages::MessageBuffer&
+operator<<(messages::MessageBuffer& msg, const quicr::Name& val)
+{
+  constexpr uint8_t size = sizeof(quicr::Name::uint_type) * 2;
+  for (size_t i = 0; i < size; ++i)
+    msg << val[size - 1 - i];
```

Comment:
thanks for testing this out. Does std::copy(..) be better here ?

Reply:
No, mostly because I don't think you could even use copy here since MessageBuffer doesn't have an iterator and the buffer is private, but also because this code should be auto-vectorised by the compiler since size is constexpr (evaluates to 16), so this should be unwrapped by the compiler to be pretty quick (and benchmarking with -O2 does show numbers that suggest that)